core/backend/src/CloudSqlite.ts

```diff
@@ -34,8 +78,8 @@ export namespace CloudSqlite {
 
   /** Properties of a CloudContainer. */
   export interface ContainerProps {
-    /** blob storage module */
-    storageType: "azure" | "google" | "aws";
+    /** blob storage provider */
+    storageType: "azure" | "google";
```

Comment:
Motivation for removal of AWS? Is it related to auto-refresh?

Reply:
it is not really supported yet. When it is, @wgoehrig  wants to call it "s3".